### PR TITLE
Add my lava level animation :^)

### PR DESCRIPTION
**Pull request by slendidev**
This PR adds a cool retro-style animation I did with schokomuffin's 3D ladyball!

It is a looping animation.

### CODE REVIEW

**Comment by frhun** [comment]
I think what this needs is for the ladyball to rotate.

**Comment by slendidev** [comment]
Good idea, sadly I no longer have the project files since I re-installed pretty much everything...

**Comment by MacDue** [comment]
So people can actually see what this is, here's the video in question: 

https://user-images.githubusercontent.com/11597044/190900891-d9e5723f-f069-4cbd-be32-54108bd4edc0.mp4

File: AUTHORS.md [hunk 1]
```diff
@@ -29,3 +29,5 @@ pieces of artwork in this repository.
 - [`music/kleines Filmröllchen - Just FLACing Around.flac`](music/kleines%20Filmröllchen%20-%20Just%20FLACing%20Around.flac) - [@kleinesfilmroellchen](https://github.com/kleinesfilmroellchen)
 - [`music/kleines Filmröllchen - Bitter Rain (Live Performance).mp3`](music/kleines%20Filmröllchen%20-%20Bitter%20Rain%20(Live%20Performance).mp3) - [@kleinesfilmroellchen](https://github.com/kleinesfilmroellchen)
 - [`music/kleines Filmröllchen - Bitter Rain.flac`](music/kleines%20Filmröllchen%20-%20Bitter%20Rain.flac) - [@kleinesfilmroellchen](https://github.com/kleinesfilmroellchen)
+- [`videos/retro_serenity0001-0120.mp4`](videos/retro_serenity0001-0120.mp4) - [@xslendix](https://github.com/xslendix)
```
Comment (MacDue) [inline AUTHORS.md]: :skeleyak: but,

Maybe come up with a slightly better name than `retro_serenity0001-0120.mp4`; that looks like what the export function of whatever you used wanted to call it.

:^)

**Review by MacDue — APPROVED** [review]
LGTM, why not add a video?

**Comment by slendidev** [comment]
> So people can actually see what this is, here's the video in question: 
> 
> https://user-images.githubusercontent.com/11597044/190900891-d9e5723f-f069-4cbd-be32-54108bd4edc0.mp4
> 
> 

You already did :^)

**Pull request merged**